simcon_project/templates/settings_view.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
i deleted this file, it no longer exists, please check your rebase.. seems like u have files that no longer exist.
I think you rebased off of branches and not off main!